crates/agglayer-rate-limiting/src/local/state/wrapper.rs

```diff
@@ -48,6 +48,7 @@ impl<S: RawState> State<S> {
     }
 
     /// Get non-modifying access to the underlying raw state.
+    #[allow(dead_code)]
```

Comment:
i think you can use `#[cfg(feature = "testutils")]` instead if only used for tests